javascript/requirejs-modules.rst

```diff
@@ -126,5 +125,5 @@ In that case, you could omit the configuration, because the entry point script i
 More information
 ================
 
-More on RequireJS' API and how to include legacy code,
+More on RequireJS API and how to include legacy code,
 which doesn't use the ``define`` module definition pattern, see the `RequireJS API documentation <https://requirejs.org/docs/api.html#define>`_.
```

Comment:
- Insert "the": "on the RequireJS"
- One sentence per line.
- s/doesn't/does not